### ISSUE TITLE
Extension modules cannot find ``spatial`` C++ header files

### ISSUE THREAD

**Issue by clinssen** [nest/nest-simulator#1790]
When building an extension module against the latest master, compilation fails because the header files in the ``spatial`` subdirectory cannot be found; or, alternatively, because they are not copied into the right place in the installation directory.

In ``include/nest/nestmodule.h``, the following lines:
```C++
// Includes from spatial:
#include "spatial/ntree.h"
#include "spatial/position.h"
```
can be replaced by
```C++
// Includes from spatial:
#include "ntree.h"
#include "position.h"
```
for an easy fix.

Questions that need to be addressed before we can move to a fix:
- Should the files be installed to a ``spatial`` subdirectory, or should the include paths be altered to remove the ``spatial`` subdirectory?
- There was discussion of reorganising the ``nestkernel`` directory, which might already make this issue go away.

Related NESTML issue: https://github.com/nest/nestml/issues/581

**Comment by hakonsbm** [comment]
@clinssen Was this fixed by #1798?

**Comment by clinssen** [comment]
Yes, thanks for pointing this out! I wrote "Fixes #1790" in a reply in that comments thread, but apparently GitHub doesn't trust that as I'm not the OP.
